Rakefile

```diff
@@ -11,11 +11,25 @@ require 'colorize'
 require 'highline'
 require 'tmpdir'
 require 'json'
+require 'zip'
 
 Cucumber::Rake::Task.new(:features) do |t|
   t.cucumber_opts = "--format pretty"
 end
 
+desc("Updates swift-doc binary with the latest version available.")
+task :swift_doc_update do
+  version = "1.0.0-beta.4"
+  system("curl", "-LO", "https://github.com/SwiftDocOrg/swift-doc/archive/#{version}.zip")
+  extract_zip("#{version}.zip", "swift-doc")
+  Dir.chdir("swift-doc/swift-doc-#{version}") do
+    system("make", "swift-doc")
+  end
+  system("cp", "swift-doc/swift-doc-#{version}/.build/release/swift-doc", "vendor/swift-doc")
```

Comment:
If someone stops the execution before the `rm -rf` command is executed, there's the risk of the zip file ending up in the repository. I suggest doing all these operations inside a temporary directory:

```
Dir.mktmpdir do |temporary_dir|
  Dir.chdir(temporary_dir) do
    # All your commands
    # When you are done, copy the binary to the right directory.
  end
end
```